src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetection.swift

```diff
@@ -106,40 +87,36 @@ final class ExposureDetection {
 
 	typealias Completion = (Result<ENExposureDetectionSummary, DidEndPrematurelyReason>) -> Void
 
-	func start(completion: @escaping Completion) {
-		Log.info("ExposureDetection: Start downloading packages.", log: .riskDetection)
-
-		self.completion = completion
-		activityState = .downloading
-		
-		downloadKeyPackages { [weak self] in
-			guard let self = self else { return }
-			Log.info("ExposureDetection: Completed downloading packages.", log: .riskDetection)
-			Log.info("ExposureDetection: Start writing packages to file system.", log: .riskDetection)
-
-			self.writeKeyPackagesToFileSystem { [weak self] writtenPackages in
-				guard let self = self else { return }
-
-				Log.info("ExposureDetection: Completed writing packages to file system.", log: .riskDetection)
-
-				self.activityState = .detecting
-
-				if let exposureConfiguration = self.exposureConfiguration {
-					if !self.deviceTimeCheck.isDeviceTimeCorrect {
-						Log.warning("ExposureDetection: Detecting summary skipped due to wrong device time.", log: .riskDetection)
-						self.endPrematurely(reason: .wrongDeviceTime)
-					} else {
-						Log.info("ExposureDetection: Start detecting summary.", log: .riskDetection)
-						self.detectSummary(writtenPackages: writtenPackages, exposureConfiguration: exposureConfiguration)
-					}
-				} else {
-					Log.error("ExposureDetection: End prematurely.", log: .riskDetection, error: DidEndPrematurelyReason.noExposureConfiguration)
-
-					self.endPrematurely(reason: .noExposureConfiguration)
-				}
-			}
-		}
-	}
+    func start(completion: @escaping Completion) {
+        Log.info("ExposureDetection: Start downloading packages.", log: .riskDetection)
+
```

Comment:
Log says downloading of packages started and after one assignment the logs say download is done. Am I missing something here?

Reply:
No you are right. Forget to remove the old logs. Removed it now.